pkg/apis/core/validation/validation.go

```diff
@@ -2954,7 +3027,7 @@ func validateEphemeralContainers(ephemeralContainers []core.EphemeralContainer,
 		idxPath := fldPath.Index(i)
 
 		c := (*core.Container)(&ec.EphemeralContainerCommon)
-		allErrs = append(allErrs, validateContainerCommon(c, volumes, idxPath, opts)...)
+		allErrs = append(allErrs, validateContainerCommon(c, volumes, podClaimNames, idxPath, opts)...)
```

Comment:
is it OK to have this in ephemeral containers?  That seems weird to me...

Reply:
I could imagine that an ephemeral container may need access to debug something related to the resource, so why not?